roles/promtail/vars/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+promtail_config_file_name: promtail.yml
+promtail_default_binary_name: promtail-{{ ansible_system | lower }}-{{ system_architecture }}
+promtail_position_file_name: positions.yml
+
+promtail_node_id:
```

Comment:
please move this group vars as it will be different for mainnet and testnet

Reply:
both promtail_node_id and promtail_node_vanity_name ?

Reply:
yeah can move the vanity name as well I guess